torch/_dynamo/eval_frame.py

```diff
@@ -148,6 +149,13 @@ def __dir__(self):
             attr for attr in super().__dir__() if attr not in orig_mod_attrs
         ]
 
+    def get_compiler_config(self):
+        # the logic should be the same to _inductor/compile_fx.py
+        from torch._inductor import config as inductor_config
+
+        with inductor_config.patch(self.dynamo_ctx.config_patches):
+            return copy.deepcopy(inductor_config._config)
```

Comment:
Add a helper method to config_utils.py rather than reading private attribute.

Reply:
added helper method get_config_copy in torch/_dynamo/config_utils.py for reading private attribute